scripts/build.mjs

```diff
@@ -0,0 +1 @@
+import '@mui/monorepo/scripts/build.mjs';
```

Comment:
Removed the custom code in our codebase based on the [`TODO` comment](https://github.com/mui/mui-x/pull/6789/files#diff-a992dd64a8c626fedc95d613f43602b91f6dc6339893afaf405dc05fa13c9ceaL1)
Checked the scripts and couldn't find any difference between codebases. 🤷

Reply:
So now it is our `babel` file that is loaded ? What made that change possible ?

Reply:
🤦 Great spot. Somehow I spotted deduplicate issue, but not this one...
It's the same problem—`getWorkspaceRoot` will return `monorepo` root and we'd need `mui-x` root.
Will revert to custom implementation and this is just another note to @michaldudak 😉

Reply:
But none the less, does not seem that we have too much needed custom code for build to happen as even not using our repo `babel.confg` it does complete 🙈

Reply:
Our babel config adds the release dates into the bundle, the core one does not
That's the main diff I have in mind